Makefile

```diff
@@ -0,0 +1,80 @@
+.PHONY: all
+all:
+	@echo 'This make file is intended for releasing only. In order to build any of Tracee's components, use their respective make files.'
```

Comment:
You have a `'` in this line which breaks the makefile. Should change the outer quotes to double quotes